src/arrayMethodJoin.test.js

```diff
@@ -9,6 +9,11 @@ test('join2 is added to [].__proto__', () => {
     .toBeInstanceOf(Function);
 });
 
+test(`join2 doesn't call default join`, () => {
+  expect([].join2.toString().includes('.join'))
+    .toBe(false);
+});
```

Comment:
it is overkill
I think we believe they won't use it

Reply:
https://github.com/mate-academy/js_array-method-push/pull/5/files

Reply:
Ok let's add the test

Reply:
@ivadimko lets check for `.join(`